internal/service/templategenerator/templategenerator_test.go

```diff
@@ -62,7 +68,6 @@ func TestGenerateModuleTemplate_Success(t *testing.T) {
 	require.Contains(t, mockFS.writtenTemplate, "moduleName: component")
 	require.Contains(t, mockFS.writtenTemplate, "component-1.0.0")
 	require.Contains(t, mockFS.writtenTemplate, "default")
-	require.Contains(t, mockFS.writtenTemplate, "test-data")
```

Comment:
Can we use a check like contains `kind: Sample` to still verify this is written?